admin/includes/css/stylesheet.css

```diff
@@ -835,3 +835,49 @@ input.faint {
 .control-label {/*for multiple inputs*/
     font-weight: 700;
 }
+
+.txt-black {
+  color:#000;
+}
+.txt-yellow {
+  color: #ff0;
+}
+.txt-orange {
+  color: #ffa500
+}
+.txt-lime {
+  color:#5ce400
+}
+.txt-red {
+  color:#f00;
+}
+.txt-navy {
+  color: #000080;
+}
+.txt-blue {
+  color: #00f;
+}
+.txt-white {
+  color:#fff;
+}
+.txt-pink {
+  color: #ffbaba;
+}
+.txt-transparent {
+  color: transparent;
+}
+.bold {
+  font-weight: bold;
+}
+.valign-middle {
```

Comment:
Could these various `.valign-` use, instead, the [Bootstrap 4](https://getbootstrap.com/docs/4.0/utilities/vertical-align/) names?  That will make it easier to migrate the admin from BS-3.

Reply:
Good plan, I had been looking for standard BS3 classes, but did not think of BS4. Will make the adjustments shortly